ggml/src/ggml.c

```diff
@@ -21064,7 +21063,7 @@ struct gguf_context * gguf_init_from_file(const char * fname, struct gguf_init_p
 
             if (ne % ggml_blck_size(info->type) != 0) {
                 fprintf(stderr, "%s: tensor '%s' of type %d (%s) number of elements (%" PRId64 ") is not a multiple of block size (%d)\n",
-                        __func__, info->name.data, (int)info->type, ggml_type_name(info->type), ne, ggml_blck_size(info->type));
+                        __func__, info->name.data, (int) info->type, ggml_type_name(info->type), ne, (int) ggml_blck_size(info->type));
```

Comment:
Instead of casting to int I think it would be better to use the `PRId64` macro like `ne` does.